server_test.go

```diff
@@ -123,7 +123,7 @@ type fakeHTTPServer struct{}
 
 func (f fakeHTTPServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {}
 
-func TestMiddleWareAuth(t *testing.T) {
+func TestMiddleWareAuthSuccess(t *testing.T) {
```

Comment:
When running `go test`, you can specify a specific test with a regex match: `go test -run TestMiddleWareAuthSuccess`. I just wanted this test to not be the root name for other tests.